### PR TITLE
Create monoMinimal.json

### PR DESCRIPTION
**Pull request by Amit-Sheen**
### New Stylesheet Submissions:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders a compiled CSS file?

### CODE REVIEW

File: src/_data/styles/monoMinimal.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  // Required
```
Comment (5t3ph) [inline src/_data/styles/monoMinimal.json:2]:
```suggestion
```

File: src/_data/styles/monoMinimal.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "title": "MonoMinimal",
+  "author": "Amit Sheen",
+  "stylesheet": "https://amitsh.com/2020/styleStage/style.css",
+  // Optional
```
Comment (5t3ph) [inline src/_data/styles/monoMinimal.json:6]:
```suggestion
```

File: src/_data/styles/monoMinimal.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "twitter": "amit_sheen",
+  "websiteTitle": "",
```
Comment (5t3ph) [inline src/_data/styles/monoMinimal.json:8]:
```suggestion
  "websiteTitle": "amitsh.com",
```

**Comment by 5t3ph** [comment]
Hi @Amit-Sheen glad to see this one in the queue!

Contrast is great, but looks like a few issues with the icons in this environment:
https://deploy-preview-35--moderncss-styles.netlify.app/styles/monominimal/

Let me know when you think that's resolved and I'll redeploy

**Comment by Amit-Sheen** [comment]
Hi @5t3ph, thanks.

I've just uploaded a new .css file to 'http://amitsh.com/2020/styleStage/style.css' and added the Font Awesome font-face so now everything should work.

Let me know if there's any more issues.

**Comment by 5t3ph** [comment]
@Amit-Sheen super close! The last thing is resolving the text not being hidden for the twitter and website links:

Screen Shot 2020-07-31 at 9 52 57 PM

**Comment by Amit-Sheen** [comment]
Sorry. In my markup I omitted the text from inside the <a> tag, so I haven't seen this problem, but it is now fixed, and I hope that it's all good. :)

New CSS file:
http://amitsh.com/2020/styleStage/style.css

**Pull request merged**